src/index.html

```diff
@@ -3,11 +3,248 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/header_logo.svg"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page" id="#">
+    <!-- #region header -->
+    <header class="header header--main">
+      <div class="top-bar">
+        <a href="#home" class="top-bar__logo"></a>
+
+        <div class="top-bar__icons">
+          <a href="tel:+1 234 555-55-55" class="icon icon--phone"></a>
+          <a href="#menu" class="icon icon--burger">
+          </a>
```

Comment:
```suggestion
          <a href="#menu" class="icon icon--burger"></a>
```

Check formatting please